pmlc/dialect/tile/builder.cc

```diff
@@ -578,7 +578,7 @@ TileBuilder::MakeProgram(StringRef name, const ProgramMutations &mutations,
   if (name.empty()) {
     name = "noname";
   }
-  IVLOG(1, "TileBuilder::MakeProgram> " << name.str());
+  IVLOG(2, "TileBuilder::MakeProgram> " << name.str());
```

Comment:
I was hoping to keep this one at 1

Reply:
Does this get called every time Keras does shape inference?  I think that might be the issue.  If not, I agree that it's high level enough to keep.

Reply:
As an example, this is called 342 times during a ResNet50 run on Intel Gen9 GPU.  Each time we print quite a lot of IR.  Log size takes a big hit:

```
$ ls -la ~/temp/logMakeProgram 
-rw-rw-r-- 1 adstraw adstraw 37026716 Sep 14 13:03 <path>/logMakeProgram

$ ls -la ~/temp/noLogMakeProgram 
-rw-rw-r-- 1 adstraw adstraw 196395 Sep 14 13:09 <path>/noLogMakeProgram
```

Reply:
OK, yes, I could see this causing lots of logs for keras. Level 2 is fine here.